src/components/DocumentationTopic/PrimaryContent/DeclarationList.vue

```diff
@@ -233,6 +233,11 @@ export default {
       margin: 0;
     }
   }
+
+  :deep(.highlighted) {
+    background: var(--color-syntax-highlighted, mark);
+    font-weight: $font-weight-semibold;
```

Comment:
As a design suggestion: I would add a transition to these properties.

Reply:
55e6fd9baec9f09aabe67afc84d9825fbfcc3b6b